src/sentry/integrations/tasks/kick_off_status_syncs_impl.py

```diff
@@ -0,0 +1,26 @@
+from sentry.models.grouplink import GroupLink
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task, retry, track_group_async_operation
+
+
+@instrumented_task(
+    name="sentry.integrations.tasks.kick_off_status_syncs_impl",
```

Comment:
We could rename this task and remove the `_impl`. We'd need a shim task with the old task name for deploy compatibility.

Reply:
`src/sentry/tasks/integrations/kick_off_status_syncs_impl.py` has a shim task with `"sentry.tasks.integrations.kick_off_status_syncs"` would that work for compatibility?

Reply:
Yeah that would work, I mostly want to avoid having `_impl` in the task name as that will require another round of renames to fix.